caproto/server/common.py

```diff
@@ -690,6 +690,8 @@ async def handle_write():
 
 
 class Context:
+    subscriptions: DefaultDict[SubscriptionSpec, Deque[Subscription]]
```

Comment:
Is it best practice to define a class var for typing purposes and then shadow it with an instance var in `__Init__`? My impulse would have been to define this with `self.subscriptions` but I am very early on the learning curve with Python typing.

Reply:
This just looks out of place, being the only thing annotated...
It's not actually defining a classvar - or any variable for that matter.  As written, it's just a type hint that says "`Context` instances have `subscriptions` which are defaultdicts".

Defining a classvar would look like the following:
```python
    subscriptions: ClassVar[DefaultDict[SubscriptionSpec, Deque[Subscription]]] = defaultdict(...)
```